tests/Main.hs

```diff
@@ -363,6 +364,20 @@ encodeCachedSubmessage = testGroup "Cached Submessages"
       FormE.toLazyByteString withImported @?=
         toLazyByteString (TPO.WithImported (Just (TPO.WithImportedPickOneWithOneof withOneof)))
 
+    testShow :: TestTree
+    testShow = testCase "Show MessageEncoding" $ do
+      let trivial = TP.Trivial 123
+          badForm = "abc"
+          encoding :: FormE.MessageEncoding TP.Trivial
+          encoding = cacheReflection trivial
+          bogus :: FormE.MessageEncoding TP.Trivial
+          bogus = FormE.unsafeByteStringToMessageEncoding badForm
+      -- Test desired behavior for valid encoding:
+      show encoding @?= "Proto3.Suite.Form.Encode.messageReflection " ++ showsPrec 11 trivial ""
```

Comment:
Use `'messageReflection`

Reply:
Actually, I kept the string literals in the test because otherwise we might not notice if the module qualification changing from an exposed module to a module that is not exposed by the library, such as `Proto3.Suite.Form.Encode.Core`.